pages/documentation/migration-V3.md

```diff
@@ -0,0 +1,822 @@
+
+We'll use an online diff service like [quickdiff.net](https://quickdiff.net/), [diffchecker.com](https://diffchecker.com), or [text-compare.com](https://text-compare.com/).
+
+Each of the files [below](#current-settings-as-of-uswds-2132) is the most current version of the USWDS default settings. For each of these files, use the diff tool to find the differences between your settings file and the current defaults.
+
+Look for instances where your project has a different value than the default. Ignore cases where a setting exists in the current version but not in your version. This indicates a new setting that probably does not apply to your project. Instances where a setting exists in your version but not in the current version can likely be ignored. This usually indicates a deprecated setting that should not affect your project, but it can be worth checking to see if that variable appears anywhere else in your codebase!
+
+When you see a setting that appears different from the current default, this is probably one of your project's custom settings. Copy this setting and add it to your new **_uswds-theme.scss** file.
+
+At the end of this process, your new **_uswds-theme.scss** file will look something like the following:
+
+<pre><code>$theme-image-path: "../uswds/img";
+$theme-font-path: "../uswds/fonts";
+$theme-show-compile-warnings: false;
+$theme-show-notifications: false;
+$theme-focus-color: "blue-50v";
+$theme-global-paragraph-styles: true;
+$theme-global-link-styles: true;
+$theme-global-content-styles: true;
+$theme-utility-breakpoints: (
+  "card": false,
+  "card-lg": true,
+  "mobile": true,
+  "mobile-lg": true,
+  "tablet": true,
+  "tablet-lg": true,
+  "desktop": true,
+  "desktop-lg": true,
+  "widescreen": true
+);</code></pre>
+
+#### Current settings (as of USWDS 2.13.2)
+
+[_uswds-theme-color.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-color.scss) → <br>
+[_uswds-theme-components.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-components.scss) → <br>
+[_uswds-theme-general.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-general.scss) → <br>
+[_uswds-theme-spacing.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-spacing.scss) → <br>
+[_uswds-theme-typography.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-typography.scss) → <br>
+[_uswds-theme-utilities.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-utilities.scss) → <br>
```

Comment:
The arrow icon is redundant here because these links have external icons attached. 

```suggestion
[_uswds-theme-color.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-color.scss) <br>
[_uswds-theme-components.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-components.scss) <br>
[_uswds-theme-general.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-general.scss) <br>
[_uswds-theme-spacing.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-spacing.scss) → <br>
[_uswds-theme-typography.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-typography.scss) → <br>
[_uswds-theme-utilities.scss](https://raw.githubusercontent.com/uswds/uswds/release-2.13.2/src/stylesheets/theme/_uswds-theme-utilities.scss) → <br>
```